tsl/test/isolation/expected/cagg_concurrent_process_wal.out

```diff
@@ -0,0 +1,27 @@
+Parsed test spec with 3 sessions
+
+starting permutation: lock_enable s1_refresh s2_refresh lock_release
+step lock_enable: 
+    SELECT debug_waitpoint_enable('multi_invalidation_process_invalidations');
+
+debug_waitpoint_enable
+----------------------
+                      
+(1 row)
+
+step s1_refresh: 
+   CALL refresh_continuous_aggregate('cagg_1', '2020-01-01 00:00:00', '2025-01-01 00:00:00');
+ <waiting ...>
+step s2_refresh: 
+   CALL refresh_continuous_aggregate('cagg_2', '2020-01-01 00:00:00', '2025-01-01 00:00:00');
+ <waiting ...>
```

Comment:
I don't doubt that the code works, but this test doesn't clearly show it. The expectation is that s1 takes the lock and blocks on the wait point while s2 blocks on the lock. But it is not possible to know that this actually happens from what one can tell here. From the output, both sessions have the same behavior. For all I know, both sessions could block on the waitpoint.

I am wondering if you can add a debug output after the lock but before the waitpoint so that it is possible to see the difference. Alternatively, show the locks held by or blocking each process.